### PR TITLE
Fix Lint Errors in Angular

### LINKED ISSUE #683: Fix Lint Errors in a generated Angular Project
If we run 'yarn lint' on a generated angular application, we run into various lint errors. We need to fix those.

### PR DESCRIPTION
**Pull request by streetcoder123**
Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/27724
Github Issue: #681

**Changes**
- It fixes various lint issues in different parts of angular application

**Note**: There is still one lint error remaining that would be handled by in another PR by @Tanya0609 as it is related to naming conventions of files.

**Procedure to check**
- Generate an Angular Application
- Inside the application install dependencies using: `yarn install`
- Run `yarn lint` to check there should be only 1 lint error

### CODE REVIEW

**Review by Tanya0609 — APPROVED** [review]
generated an angular project with all four type pages selected. Generated app works as usual. 
I did yarn install and yarn lint. All of the lint errors are gone except one that will be addressed in a different PR.

File: templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListForm/list-form.component.ts [hunk 1]
```diff
@@ -5,20 +5,15 @@
...
+  textField: string;
   @Output() inputTextEventEmitter = new EventEmitter<string>();
-  get textField() {
-      return this._textField;
-  }
-  set textField(textInput) {
-      this._textField = textInput;
-  }
+
   constructor() { }
```
Comment (jsondoo) [inline templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListForm/list-form.component.ts:11]: How come this closing bracket isn't on a new line like the other ones? Or is constructor a special case

**Review by jsondoo — APPROVED** [review]
Verified lint is passing

File: templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListForm/list-form.component.ts (hunk 1 above)
Comment (streetcoder123) [reply templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListForm/list-form.component.ts:11]: I don't think there is something special with constructor. But, this is normally the pattern I see for methods with no content.

File: templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListItem/list-item.ts [hunk 2]
```diff
@@ -5,6 +5,7 @@
   templateUrl: './list-item.component.html'
 })
 export class ListItemComponent implements OnInit {
+  // tslint:disable-next-line
```
Comment (jcoc611-microsoft) [inline templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListItem/list-item.ts:8]: Quick question, why is this needed?

Comment (streetcoder123) [reply templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListItem/list-item.ts:8]: Good question! So, the backend returns a list item that has attributes `text` and `_id`. When we make HTTP request to get the list data from the backend, it is assumed to be of type 'IListItem'. We cannot/should not change the data returned from the backend and if we modify the 'IListItem' in front-end to have `text` & `id` instead of `text` & `_id`, the data returned from the backend would break.

So, in that case, I thought we should keep the attribute `_id` as is and disable the lint for this one. Do you have any other suggestions?

Comment (jcoc611-microsoft) [reply templates/Web/Pages/Angular.List/src/app/app-shell/AngularList/ListItem/list-item.ts:8]: It sounds reasonable!

**Review by jcoc611-microsoft — APPROVED** [review]
